tests/integration/compilation/test_collector_compilation_test.py

```diff
@@ -0,0 +1,34 @@
+from pathlib import Path
+
+from pytest_mock import MockerFixture
+
+from protostar.utils.compiler.pass_managers import (
+    ProtostarPassMangerFactory,
+    TestCollectorPassManagerFactory,
+)
+from protostar.utils.starknet_compilation import CompilerConfig, StarknetCompiler
+
+
+def expected_collected_functions_oracle(path: Path):
```

Comment:
oracle? What does it mean?

Reply:
https://en.wikipedia.org/wiki/Test_oracle